release-pipeline.yml

```diff
@@ -5,6 +5,13 @@ trigger:
 
 pr: none # disable CI build for PR
 
+resources:
+  repositories:
+  - repository: BedrockHomebrewCore # The name used to reference this repository in the checkout step
+    type: github
+    endpoint: "bedrock-cli"
+    name: microsoft/homebrew-bedrock
```

Comment:
Should we rename this to have `bedrock` be the prefix? ie. `bedrock-homebrew`?

Also, what permissions are needed to protect that repository?  Looks like the pipeline is commiting and pushing directly to master, do we utilize a PAT or some other service connection in the pipeline?

Reply:
`homebrew-bedrock` is the convention from homebrew. This allows us to subscribe to a tap like this: `brew tap microsoft/bedrock`. More information on third party tap repos is available here: [https://docs.brew.sh/Taps](https://docs.brew.sh/Taps)

"On GitHub, your repository must be named homebrew-something in order to use the one-argument form of brew tap. The prefix ‘homebrew-‘ is not optional. (The two-argument form doesn’t have this limitation, but it forces you to give the full URL explicitly.)"

Reply:
I'll need to revisit permissions, on the test, I didn't set any specific restrictions to the test repo `edaena/homebrew-core` which is analogous to `microsoft/homebrew-bedrock`

Reply:
Got it. Thanks.